### PR TITLE
UI Changes and wider search

### PR DESCRIPTION
**Pull request by sahal-mulki**
Ok, so I wish to contribute to this project.
I made some minor changes, here they are:

1. Cap tweet preview for the folium marker preview to 100 letters.
2. Add one more hashtag for tweepy to search (#OSINT).
3. Adjust default coordinates and zoom.
4. Change tileset to something more fitting (in my opinion).
5. Change User Agent which geopy uses for Nominatim.

Please inform me of anything wrong I did (this is my first pr).

### CODE REVIEW

File: plot.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
 class Map:
     def __init__(self,tweets_file, dist_dir):
         self.tweets_file = tweets_file
-        self.geolocator = Nominatim(user_agent="map ukraine")
-        self.m = folium.Map(location = (48.3794, 31.1656), zoom_start=6)
+        self.geolocator = Nominatim(user_agent="LiveActionMap")
+        self.m = folium.Map(location=[49.422181441686966, 32.04020925216723], zoom_start=5.5, tiles="Stamen Toner")
```
Comment (kinshukdua) [inline plot.py:19]: Google tells me the coordinates are 48.3794, 31.1656. Any  source for the location? Plus changing the tiles changes the text to English which is something we don't want. I like the idea of a new user_agent and the summary. Can you remove the folium.Map line and I'll merge the other changes.

**Review by kinshukdua — CHANGES_REQUESTED** [review]

File: plot.py (hunk 1 above)
Comment (sahal-mulki) [reply plot.py:19]: What if I change the coords and tiles but keep the zoom, is that agreeable?

Comment (sahal-mulki) [reply plot.py:19]: Or this tileset, which is aesthetically pleasing and using the native language?

Comment (sahal-mulki) [reply plot.py:19]: https://leaflet-extras.github.io/leaflet-providers/preview/#filter=Stadia.AlidadeSmoothDark

Comment (sahal-mulki) [reply plot.py:19]: nvm its paid

Comment (kinshukdua) [reply plot.py:19]: Zoom level is an int, zoom=5.5 defaults to 6. It's also okay to keepthe default tile.

Comment (sahal-mulki) [reply plot.py:19]: k

**Comment by sahal-mulki** [comment]
Changes are done now :)

File: plot.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ class Map:
     def __init__(self,tweets_file, dist_dir):
         self.tweets_file = tweets_file
         self.geolocator = Nominatim(user_agent="LiveActionMap")
-        self.m = folium.Map(location=[49.422181441686966, 32.04020925216723], zoom_start=5.5, tiles="Stamen Toner")
+        self.m = folium.Map(location=[49.422181441686966, 32.04020925216723], zoom_start=5.5, min_zoom=5.5)
```
Comment (kinshukdua) [inline plot.py:20]: Can you leave the default map as it is?

Comment (sahal-mulki) [reply plot.py:20]: So no min zoom?

Comment (kinshukdua) [reply plot.py:20]: Nope

Comment (sahal-mulki) [reply plot.py:20]: got it

Comment (sahal-mulki) [reply plot.py]: Done.

**Comment by sahal-mulki** [comment]
Changes done

**Comment by kinshukdua** [comment]
> Changes done

Can you copy paste the self.m = folium.Map from main (this github repository)? You've changed this line.

**Comment by kinshukdua** [comment]
> ok

Last nitpick, can you set the zoom level back to 6?

**Comment by sahal-mulki** [comment]
ohk

**Comment by sahal-mulki** [comment]
Done

**Comment by kinshukdua** [comment]
Thanks @sahal-mulki and congratulations on your first PR getting merged !!

**Comment by sahal-mulki** [comment]
thank YOU.

**Pull request merged**